### PR TITLE
Fix telegraf health check with internal plugin enabled

### PR DESCRIPTION
**Pull request by nsteinmetz**
* Fix livenessProbe, readynessProbe and health service
* remove @naseemkullah from telegraf(-ds) maintainers list as requested on slack

### CODE REVIEW

File: charts/telegraf/README.md [hunk 1]
```diff
@@ -39,6 +39,30 @@ helm uninstall telegraf
...
 The command removes all the Kubernetes components associated with the chart and deletes the release.
 
+## livenessProve, readinessProbe and health service
+
+Be sure to have at least these inputs and outputs enabled so that readinessProbe and livenessProbe can work as expected and that the health service endpoint is correctly set up:
+
+```yaml
+[...]
+
+config:
```
Comment (rawkode) [inline charts/telegraf/README.md:49]: Can't we enforce this?

File: charts/telegraf/values.yaml [hunk 2]
```diff
@@ -122,6 +122,8 @@ config:
           - "http://influxdb.monitoring.svc:8086"
         database: "telegraf"
   inputs:
+    - internal:
```
Comment (rawkode) [inline charts/telegraf/values.yaml:125]: and this :+1:

File: charts/telegraf/README.md (hunk 1 above)
Comment (nsteinmetz) [reply charts/telegraf/README.md]: Indeed, we can - I thought we could not when I first implemented it ;-)

File: charts/telegraf/values.yaml (hunk 2 above)
Comment (nsteinmetz) [reply charts/telegraf/values.yaml]: Yep, same comment as previous one.

File: charts/telegraf/README.md (hunk 1 above)
Comment (nsteinmetz) [reply charts/telegraf/README.md]: done

**Comment by Disturbing** [comment]
Wow great work guys :) I am literally over here in Hong Kong debugging this issue, and you guys are about to approve this in.

**Comment by rawkode** [comment]
Thanks @nsteinmetz, great work (as always!)

**Pull request merged**